src/lang/Messages.properties

```diff
@@ -737,6 +737,7 @@ Core options:\n\
 \t-port <port>             Overrides the port used for proxying specified in the configuration file\n\
 \t-lowmem                  Use the database instead of memory as much as possible - this is still experimental\n\
 \t-experimentaldb          Use the experimental generic database code, which is not surprisingly also still experimental\n\
+\t-nostdout                Disables the default logging through the standard output\n\
```

Comment:
It's not technically wrong but it reads a little awkwardly. I'd suggest removing the second "the":
`Disables the default logging through standard output`